Index: data-management/src/main/java/com/bonus71/data/ResourceModifier.java
===================================================================
diff --git a/data-management/src/main/java/com/bonus71/data/ResourceModifier.java b/data-management/src/main/java/com/bonus71/data/ResourceModifier.java
deleted file mode 100644
--- a/data-management/src/main/java/com/bonus71/data/ResourceModifier.java	(revision 14e4d8ae63e9ca22da2cb410500edd73d049d945)
+++ /dev/null	(revision 14e4d8ae63e9ca22da2cb410500edd73d049d945)
@@ -1,11 +0,0 @@
-package com.bonus71.data;
-
-/**
- * ResourceModifier class.
- *
- * @author jskarpetis
- * @version 1.0
- * Created on 19/11/25.
- */
-public class ResourceModifier {
-}
Index: data-management/src/main/java/com/bonus71/data/DatabaseUploader.java
===================================================================
diff --git a/data-management/src/main/java/com/bonus71/data/DatabaseUploader.java b/data-management/src/main/java/com/bonus71/data/DatabaseUploader.java
deleted file mode 100644
--- a/data-management/src/main/java/com/bonus71/data/DatabaseUploader.java	(revision 14e4d8ae63e9ca22da2cb410500edd73d049d945)
+++ /dev/null	(revision 14e4d8ae63e9ca22da2cb410500edd73d049d945)
@@ -1,11 +0,0 @@
-package com.bonus71.data;
-
-/**
- * DatabaseUploader class.
- *
- * @author jskarpetis
- * @version 1.0
- * Created on 19/11/25.
- */
-public class DatabaseUploader {
-}
Index: data-management/src/main/java/com/bonus71/data/ResourceViewer.java
===================================================================
diff --git a/data-management/src/main/java/com/bonus71/data/ResourceViewer.java b/data-management/src/main/java/com/bonus71/data/ResourceViewer.java
deleted file mode 100644
--- a/data-management/src/main/java/com/bonus71/data/ResourceViewer.java	(revision 14e4d8ae63e9ca22da2cb410500edd73d049d945)
+++ /dev/null	(revision 14e4d8ae63e9ca22da2cb410500edd73d049d945)
@@ -1,11 +0,0 @@
-package com.bonus71.data;
-
-/**
- * ResourceViewer class.
- *
- * @author jskarpetis
- * @version 1.0
- * Created on 19/11/25.
- */
-public class ResourceViewer {
-}
Index: data-management/src/main/java/com/bonus71/data/ResourceReader.java
===================================================================
diff --git a/data-management/src/main/java/com/bonus71/data/ResourceReader.java b/data-management/src/main/java/com/bonus71/data/ResourceReader.java
deleted file mode 100644
--- a/data-management/src/main/java/com/bonus71/data/ResourceReader.java	(revision 14e4d8ae63e9ca22da2cb410500edd73d049d945)
+++ /dev/null	(revision 14e4d8ae63e9ca22da2cb410500edd73d049d945)
@@ -1,33 +0,0 @@
-package com.bonus71.data;
-
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-
-public class ResourceReader {
-
-    public List<List<String>> readCSVAsList(String fileName) throws IOException {
-        List<List<String>> data = new ArrayList<>();
-
-        InputStream inputStream = getClass().getClassLoader().getResourceAsStream(fileName);
-
-        if (inputStream == null) {
-            throw new IOException("File not found in resources: " + fileName);
-        }
-
-        try (BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream))) {
-            String line;
-            while ((line = reader.readLine()) != null) {
-                String[] values = line.split(",");
-                List<String> row = new ArrayList<>(Arrays.asList(values));
-                data.add(row);
-            }
-        }
-
-        return data;
-    }
-}
